src/System.Management.Automation/engine/hostifaces/PowerShellProcessInstance.cs

```diff
@@ -83,6 +88,10 @@ public PowerShellProcessInstance(Version powerShellVersion, PSCredential credent
             processArguments = string.Format(CultureInfo.InvariantCulture,
                 "{0} -s -NoLogo -NoProfile", processArguments);
 
+#if CORECLR
+            processArguments = " -s -NoLogo -NoProfile";
+#endif 
```

Comment:
```
          processArguments = string.Format(CultureInfo.InvariantCulture,  
                   "{0} -s -NoLogo -NoProfile", processArguments);  
#if CORECLR  
          processArguments = " -s -NoLogo -NoProfile";  
#endif
```
it's better to not have unused code in CoreCLR build. Can you please make it like the following?

```
#if CORECLR  
           string processArguments = " -s -NoLogo -NoProfile";
#elif
           // Adding Version parameter to powershell.exe 
           // Version parameter needs to go before all other parameters because the native layer looks for Version or  
           // PSConsoleFile parameters before parsing other parameters. 
           // The other parameters get parsed in the managed layer. 
           Version tempVersion = powerShellVersion ?? PSVersionInfo.PSVersion; 
           string processArguments = string.Format(CultureInfo.InvariantCulture, 
                      "-Version {0}", new Version(tempVersion.Major, tempVersion.Minor)); 
  
           processArguments = string.Format(CultureInfo.InvariantCulture,  
                      "{0} -s -NoLogo -NoProfile", processArguments);  
#endif   
```